deployment/Makefile

```diff
@@ -7,8 +7,7 @@ CAT_CMD=$(if $(filter $(OS),Windows_NT),type,cat)
 OPERATOR_VERSION_FILE=./OPERATOR_VERSION
 OPERATOR_VER=v$(shell $(CAT_CMD) $(OPERATOR_VERSION_FILE))
 OPERATOR_VER_NUM=$(shell $(CAT_CMD) $(OPERATOR_VERSION_FILE))
-CONTROLLER_VERSION_FILE=../CONTROLLER_VERSION
-CONTROLLER_VER=v$(shell $(CAT_CMD) $(CONTROLLER_VERSION_FILE))
+CONTROLLER_VER=$(shell curl -s https://pkg.go.dev/github.com/project-codeflare/multi-cluster-app-dispatcher | grep -o 'v[0-9]\+\.[0-9]\+\.[0-9]\+' | head -n 1)
```

Comment:
This logic seems suspect. I thought the controller version would be derived from a git TAG.

Reply:
Not sure how the operator build triggered, but looking at the docker hub registry **darroyo/mcad-operator** seems its been long build is triggered..

Reply:
Yes correct the logic here is to retrieve latest release version from **mcad pkg.go.dev** , this wont work in case of patch build.. 
Probably either we can pass this as property when triggering the build or else change this logic to get latest git tag will be the best approach i guess

Reply:
For OLM tests I retrieve the latest tag using `git describe --tags --abbrev=0`

Reply:
Also i am thinking do we need similar approach here https://github.com/project-codeflare/multi-cluster-app-dispatcher/blob/1d7cbee5b42857e35c45df39698373929051aea0/Makefile#L26

Reply:
Possibly that should be updated too, though others will know better